### PR TITLE
feat: adds stable diffusion notebook with PyTorch serving

### PR DESCRIPTION
**Pull request by telpirion**
This notebook has been reviewed previously: https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/1491

### CODE REVIEW

**Comment by andrewferlitsch** [comment]
Failure:
TypeError: Descriptors cannot not be created directly.
Step #5: Step #4: If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.

File: notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb [hunk 1]
```diff
@@ -0,0 +1,979 @@
+{
```
Comment (gericdong) [inline notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: <p>The links need to be updated for the new location.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (gericdong) [inline notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: <p>Typo: estimage &gt; estimate</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (gericdong) [inline notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: <p>Typo: everying &gt; everything</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (gericdong) [inline notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: <p>"Stable Diffusion 2.0 model" - should it be a Stable Diffusion 1.5 model used in this tutorial?</p><p>" three step process" - There are four steps listed below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (gericdong) [inline notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: <p>Is the cleaning up section missing? Should we have some cleanup steps here, e.g. undeploying and deleting resources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Comment (telpirion) [reply notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: Done.

Comment (telpirion) [reply notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: Done.

Comment (telpirion) [reply notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: Done.

Comment (telpirion) [reply notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: Done.

Comment (telpirion) [reply notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb:1]: Ugh! I had one, but it must've somehow gotten deleted. Added back.

**Review by gericdong — APPROVED** [review]
LGTM

**Pull request merged**